subsys/partition_manager/pm.yml.trustzone

```diff
@@ -0,0 +1,8 @@
+# Create a span for the RAM to be configured as SECURE by the spm.
+sram_secure:
+  span: []
+  region: sram_primary
+
+sram_nonsecure:
+  span: [sram_primary]
```

Comment:
Should we add a comment here explaining what this does?

It actually looks quite confusing spanning the same label as the region name, and from the top of my head I'm unable to see why its needed, but I guess it is :D

Reply:
sram_primary is the dynamic partition.

Reply:
IMO a comment explaining this would be good, since the region has the same name as the dynamic partition and this can be confusing when reading this configuration file

Reply:
Done